### DISCUSSION TITLE
What is the best way to use dreambooth at the moment

### DISCUSSION THREAD

**Discussion by vitalijmirnyj** [TheLastBen/fast-stable-diffusion#2187] [Q&A]
Hello everyone :) I have a question regarding training my own model. I used to use google colab, but it stopped working for some reason. As i understand i need to upgrade to pro plan. Is this correct? Also is there a tutorial somewhere how to use dreambooth in the runpod? I do not mind paying money for using dreambooth and would be very greatful for anyone answering how to use dreambooth at the moment. Thank you :)

**Comment by TheLastBen** [comment]
just follow this link, https://www.runpod.io/console/gpu-browse?template=runpod-stable-unified, pick a 16gb cheap GPU and deploy the pod, it will contain 3 notebooks, one for A1111, one for DBv1 and one for DBv2, they work exactly like the colab notebooks.

remember to stop the pod once you're done working with it

**Reply by vitalijmirnyj** [reply]
Thank you!
